data/json/items/armor/suits_protection.json

```diff
@@ -224,6 +224,750 @@
     "name": { "str": "XL plate armor" },
     "copy-from": "armor_lightplate",
     "proportional": { "weight": 1.125, "volume": 1.13, "price": 1.25 },
+    "flags": [ "VARSIZE", "OUTER", "STURDY", "OVERSIZE" ]
+  },
+  {
+    "id": "armor_lc_lightplate",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "mild steel 2 mm plate armor" },
+    "description": "A light suit of plate armor with a 2 mm thick chestpiece.  This one has been made with mild steel.",
+    "weight": "15 kg",
+    "volume": "17500 ml",
+    "price": 60000,
+    "price_postapoc": 12000,
+    "to_hit": -5,
+    "bashing": 8,
+    "material": [ { "type": "low_steel", "portion": 9 }, { "type": "leather", "portion": 1 } ],
+    "symbol": "[",
+    "looks_like": "armor_larmor",
+    "color": "light_gray",
+    "warmth": 20,
+    "longest_side": "60 cm",
+    "material_thickness": 2,
+    "flags": [ "VARSIZE", "OUTER", "STURDY" ],
+    "armor_data": {
+      "armor": [
+        {
+          "material": [
+            { "type": "low_steel", "covered_by_mat": 100, "thickness": 2.0 },
+            { "type": "leather", "covered_by_mat": 5, "thickness": 1.0 }
+          ],
+          "covers": [ "torso" ],
+          "coverage": 100,
+          "encumbrance": 15
+        },
+        {
+          "material": [
+            { "type": "low_steel", "covered_by_mat": 100, "thickness": 1.0 },
+            { "type": "leather", "covered_by_mat": 5, "thickness": 1.0 }
+          ],
+          "covers": [ "arm_l", "arm_r" ],
+          "coverage": 100,
+          "encumbrance": 10
```

Comment:
keep in mind this value is halved in game because of "VARSIZE" so these are all better than current plate armor though I see that they also have reduced thickness.